src/saturn_engine/worker/executors/arq/executor.py

```diff
@@ -81,15 +87,76 @@ def deserialize(self, data: bytes) -> dict[str, t.Any]:
         obj = pickle.loads(data)  # noqa: S301
         return obj
 
+    def create_connection_pool(self) -> ConnectionPool:
+        # Create a connection from
+        redis_settings = RedisSettings.from_dsn(self.options.redis_url)
+        kwargs: dict[str, t.Any] = {
```

Comment:
Most of this code is taken from `Redis` init that do the same but with the default, non-blocking pool